stories/plotly/plotly-pie.stories.js

```diff
@@ -42,4 +44,15 @@ export const NoLabels = Template.bind({})
 NoLabels.args = { showPercentage: false }
 
 export const CustomColors = Template.bind({})
-CustomColors.args = { customColors: ['#88CCEE', '#CC6677', '#DDCC77', '#477733', '#0F2288'] }
+CustomColors.args = { 
+  customColors: {
+    color1: ['#88CCEE', '#CC6677', '#DDCC77', '#477733', '#0F2288']
```

Comment:
`customColors `always has 2 keys, color1 & color2. So let's put some values for color2 as well.